client/app/src/test/java/com/example/criengine/MyBooksAdapterTest.java

```diff
@@ -0,0 +1,135 @@
+package com.example.criengine;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.test.core.app.ApplicationProvider;
+
+import com.example.criengine.Adapters.MyBooksAdapter;
+import com.example.criengine.Database.DatabaseWrapper;
+import com.example.criengine.Objects.Book;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.ArrayList;
+
+@RunWith(RobolectricTestRunner.class)
+public class MyBooksAdapterTest {

```

Comment:
PLZ have javadocs at the start of every file (class header) and also above each method. I think you got most of the docs for the methods but you're missing for the class header